test/verify/check-storage-vdo

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+
+from testlib import *
+from storagelib import *
+
+class TestStorage(StorageCase):
+    def testVdo(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        if m.execute("which vdo || true") == "":
```

Comment:
Should this be a decorator?

Reply:
Hmm, the test actually checks that the VDO panel does not appear on images that don't have the vdo utilities.  We do this for other optional features as well, such as iSCSI.

Reply:
Right, i didn't mean to skip. I just saw it was repeated everywhere, but probably not a big deal.